articles/connections/references/apple-native/guides/add-siwa-to-web-app.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Add Sign In with Apple to Your Web App
+description: Learn how to add native login functionality to your web app with Apple. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Your Web App
+
+To set up and configure native Sign In with Apple, you will do the following:
+
+* [Complete the prerequisites](#prerequisites): Have an Apple Developer account (paid), an Auth0 tenant set up with a custom domain, and a web application configured to use Auth0 for authentication at that domain.  
+* [Register your app in your Apple Developer account](#register-your-app-in-your-apple-developer-account): Obtain Apple IDs and keys for the application connection settings in the Dashboard.
+* [Create your web app](#create-your-web-application): Including verifying your domain ownership with Apple and configuring Apple's email relay service, if necessary.
+* [Configure the application connection in Auth0](#configure-and-test-the-connection-in-Auth0): Complete the social connection settings configuration information with the IDs and keys you obtained for your application from Apple. 
+
+## Prerequisites
+
+<%= include('../_includes/_siwa-prerequisites') %>
+
+## Register your web app in your Apple Developer account
+
+Once your Apple Developer account is set up, follow the instructions in the [Apple Setup Guide](/connections/apple-setup) to set up your application. 
+
+When setting up your application, make sure you save the following IDs and keys for the application connection settings in the Auth0 Dashboard:
+
+* Client ID (the Services ID) 
+* AppID
+
+1. If you haven't already done so, crate an [Apple Developer Account](https://developer.apple.com/)
+
+2. When you sign into the developer account, go to the [Certificates, IDs, & Profiles](https://developer.apple.com/account/resources/certificates/list) section, and choose the [Identifiers](https://developer.apple.com/account/resources/identifiers/list/serviceId) subsection on the left menu.
+
+3. Click **Register an App ID** button.
```

Comment:
We're in web app, so not App ID, Service ID

Reply:
Do you mean that you select "Register a Service ID" ?